e2e-tests/commands/insurance/eligibility/forms.js

```diff
@@ -38,8 +45,24 @@ export const completeCompanyDetailsForm = () => {
   submitButton().click();
 };
 
-export const completeInsuredAmountForm = () => {
-  selectRadioAndSubmit(FIELD_VALUES.NO);
+/**
+ * completeTotalValueInsuredForm
+ * completes and submits the total value insured form
+ * either selects above or below 250k based on param
+ * @param {Boolean} aboveUpperValue - if above or below 250k radio should be selected
+ */
+export const completeTotalValueInsuredForm = ({ aboveUpperValue = true }) => {
+  const fieldOptionsAbove = FIELDS_ELIGIBILITY[TOTAL_CONTRACT_VALUE].OPTIONS.ABOVE;
+  const fieldOptionsBelow = FIELDS_ELIGIBILITY[TOTAL_CONTRACT_VALUE].OPTIONS.BELOW;
+
+  let fieldId = `${TOTAL_CONTRACT_VALUE}-${fieldOptionsBelow.ID}`;
+
+  if (aboveUpperValue) {
+    fieldId = `${TOTAL_CONTRACT_VALUE}-${fieldOptionsAbove.ID}`;
```

Comment:
Can we destructure the options so we can do `ABOVE.ID` etc, instead of having 2 extra variables on lines 55/56?

Could we also:
- Rename the `aboveUpperValue` param to be `secondOption`? Feels more generic, i.e if we add another radio or rename the current one, `aboveUpperValue` would need renaming
- Rename the function/command to include `completeAndSubmit`?